app/scripts/api.js

```diff
@@ -34,6 +34,14 @@ const createApi = function api(context, pubSub) {
     publish: apiPubSub.publish,
     // Public API
     public: {
+      /**
+       * HiGlass version
+       * @return {string} Version number
+       */
+      get version() {
+        return VERSION;
```

Comment:
Can we get a comment explaining where VERSION is originally defined?

Reply:
Good point! Will add.